src/CMakeLists.txt

```diff
@@ -584,7 +584,7 @@ endif(${WITH_LTTNG})
 
 add_subdirectory(global)
 
-if(NOT WIN32)
+if(WITH_CEPHFS)
```

Comment:
:+1: for minimizing the dependencies. The `WITH_CEPHFS && WIN32` case can't happen, right?

Reply:
see see https://github.com/ceph/ceph/blob/becbe5fc4a72a5ebd97b04e30a1f5c17080dd889/win32_build.sh#L162. moreover, i think we should not hardwire a feature to a platform unless that feature is platform specific.